shared/services/config/stader-config.go

```diff
@@ -159,20 +161,30 @@ func LoadFromFile(path string) (*StaderConfig, error) {
 	if err := yaml.Unmarshal(configBytes, &settings); err != nil {
 		return nil, fmt.Errorf("could not parse settings file: %w", err)
 	}
+	// to be removed : start-01
+	// - remove encoding/json dependency
+	b, err := json.MarshalIndent(settings, "", "  ")
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+	fmt.Print(string(b))
+	// to be removed : end
 
 	// Deserialize it into a config object
-	cfg := NewStaderConfig(filepath.Dir(path), false)
+	cfg := NewStaderConfig(filepath.Dir(path), false, false)
 	err = cfg.Deserialize(settings)
 	if err != nil {
 		return nil, fmt.Errorf("could not deserialize settings file: %w", err)
 	}
-
+	// to be removed : start-02
+	fmt.Println("isSSVMode : ", cfg.IsSSVMode)
+	// to be removed : end
 	return cfg, nil
 
 }
 
 // Creates a new Stader configuration instance
-func NewStaderConfig(staderDir string, isNativeMode bool) *StaderConfig {
```

Comment:
Do not changes this API

Reply:
any reason ?

Reply:
To keep it simple, do not treat new flag as special.

Reply:
ssv mode flag is set in stader config, if we dont add this is NewStaderConfig func, we break some calls of this method in other places.